src/Begin.f90

```diff
@@ -87,135 +87,135 @@ Subroutine BEGIN(param_file,spatial_file)
+!     the river mile of the headwaters.
+!
+  read(90,'(i5,11x,i4,10x,i5,15x,i5,15x,f10.0,i5)') no_cells(nr) &
       ,head_name,trib_cell,main_stem,rmile0
-!
-!     If this is reach that is tributary to cell TRIB_CELL, give it the
-!     pointer TRIB(TRIB_CELL) the index of this reach for further use.
-!     Also keep track of the total number of tributaries for this cell
-!
+!
+!     If this is reach that is tributary to cell TRIB_CELL, give it the
+!     pointer TRIB(TRIB_CELL) the index of this reach for further use.
+!     Also keep track of the total number of tributaries for this cell
+!
   if (trib_cell.gt.0) then
-    no_tribs(trib_cell)=no_tribs(trib_cell)+1
-    trib(trib_cell,no_tribs(trib_cell))=nr
-  end if
-!
-!     Reading Mohseni parameters for each headwaters (UW_JRY_2011/06/18)
-!
-  read(90,*) alphaMu(nr),beta(nr) &
-            ,gmma(nr),mu(nr),smooth_param(nr)
-!
-!     Reading Reach Element information
-!
-  first_cell=.true.
-  do nc=1,no_cells(nr)
-    ncell=ncell+1
-!
-!   Read the data for point sources
-!
-    if (source) then
+    no_tribs(trib_cell)=no_tribs(trib_cell)+1
+    trib(trib_cell,no_tribs(trib_cell))=nr
+  end if
+!
+!     Reading Mohseni parameters for each headwaters (UW_JRY_2011/06/18)
+!
+  read(90,*) alphaMu(nr),beta(nr) &
+            ,gmma(nr),mu(nr),smooth_param(nr)
+!
+!     Reading Reach Element information
+!
+  first_cell=.true.
+  do nc=1,no_cells(nr)
+    ncell=ncell+1
+!
+!   Read the data for point sources
+!
+    if (source) then
 !
 !  Place holder for point source input
-!
-    end if 
-!
-!     The headwaters index for each cell in this reach is given
-!     in the order the cells are read
-!
-!     Card Type 3. Cell indexing #, Node # Row # Column Lat Long RM
-!
-!     Variable ndelta read in here.  At present, number of elements
-!     is entered manually into the network file (UW_JRY_2011/03/15)
-!
-    read(90,'(5x,i5,5x,i5,8x,i5,6x,a8,6x,a10,7x,f10.0,i5)')  &
-              node,nrow,ncol,lat,long,rmile1,ndelta(ncell)

-!
-!    Set the number of segments of the default, if not specified
-!
-    if (ndelta(ncell).lt.1) ndelta(ncell)=n_default
-    if(first_cell) then
-      first_cell=.false.
-      head_cell(nr)=ncell
-      x_dist(nr,0)=5280.*rmile0
+!
+    end if 
+!
+!     The headwaters index for each cell in this reach is given
+!     in the order the cells are read
+!
+!     Card Type 3. Cell indexing #, Node # Row # Column Lat Long RM
+!
+!     Variable ndelta read in here.  At present, number of elements
+!     is entered manually into the network file (UW_JRY_2011/03/15)
+!
+    read(90,'(5x,i5,5x,i5,8x,i5,6x,a8,6x,a10,7x,f10.0,i5)')  &
+              node,nrow,ncol,lat,long,rmile1,ndelta(ncell)
+!
+!    Set the number of segments of the default, if not specified
+!
+    if (ndelta(ncell).lt.1) ndelta(ncell)=n_default
+    if(first_cell) then
+      first_cell=.false.
+      head_cell(nr)=ncell
+      x_dist(nr,0)=5280.*rmile0
```

Comment:
Don't hardcode constants. Declare a constant: `feet_per_mile` (if that is what this is) and then use that consistently throughout the code. It would be best if all constants were collected in one place.